docs/src/components/Theming/TokenList.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { useTheme } from '@aws-amplify/ui-react';
+import { isDesignToken } from '@aws-amplify/ui';
+import {
+  SpaceBlock,
+  BorderWidthBlock,
+  FontFamilyBlock,
+  FontSizeBlock,
+  LineHeightBlock,
+  FontWeightBlock,
+  ColorBlock,
+  RadiusBlock,
+} from './TokenBlocks';
+
+type Namespaces =
+  | 'colors'
+  | 'radii'
+  | 'space'
+  | 'borderWidths'
+  | 'lineHeights'
+  | 'fonts'
+  | 'fontSizes'
+  | 'fontWeights';
+
+export function createTokenList(tokens) {
```

Comment:
Suggest moving in to a _components/Theming/utils.ts_ file, and adding strict-ish typing to make reading (and future refactoring if needed) easier

Reply:
I ended up moving this to a utils file, though typing could use some work (see [this comment](https://github.com/aws-amplify/amplify-ui/pull/2023#discussion_r895903948) here)